src/test/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolderTest.java

```diff
@@ -319,7 +319,7 @@ public void runningWorflowJobBuildWithAbortBuildsOption() throws Exception {
         folder.kids.add("kid");
         folder.recompute(Result.SUCCESS);
         folder.assertItemNames(1, "kid");
-        folder.getItem("kid").setDefinition(new CpsFlowDefinition("node { sleep(Long.MAX_VALUE) }", true));
+        folder.getItem("kid").setDefinition(new CpsFlowDefinition("semaphore('foo')", true));
```

Comment:
This pairs with `SemaphoreStep.waitForStart`.

Reply:
Done